### DISCUSSION TITLE
Error TensorFlow on Image Generator

### DISCUSSION THREAD

**Discussion by ArijayaAzis** [teamalgoritma/community#932] [Q&A] [answered]
Hallo tim,

Saya ada error ketika menjalankan image Generator untuk validasi data
![image](https://github.com/user-attachments/assets/82d1f17d-f214-44cd-849b-e463f73c038a)

pada console menghasilkan error seperti ini
![image](https://github.com/user-attachments/assets/83822638-afbc-45b4-a54d-c388aa704f6b)

dengan keterangan:
```
2024-12-05 18:52:03.400950: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: SSE4.1 SSE4.2 AVX AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
```

---
Saya cari di stackoverflow, mayoritas karena device macbook dan M1 / M2, tetapi saya belum menemukan untuk intel

**Comment by hwulanayu** [comment, marked as answer]
Dear @ArijayaAzis,

Bapak dapat mencoba hal-hal berikut:

1. Upgrade tensorflow library
`pip install --upgrade tensorflow`
3. Gunakan tensorflow khusus Intel, bukan Apple chip
`pip install intel-tensorflow`
4. **Opsional**: Jika langkah di atas sudah dilakukan tetapi belum berhasil, Bapak dalam melakukan unistall dan install kembali
`pip uninstall tensorflow`
`pip uninstall intel-tensorflow`
5. Pastikan xcode terinstall dengan baik
`xcode-select --install`

Jika Bapak sudah melakukan hal tersebut tetapi masih belum berhasil, silakan reply kembali discussion ini.

Best Regards,
Team Algoritma

**Reply by IrfanChairurrachman** [reply]
Dear @ArijayaAzis ,

Terima kasih telah bertanya melalui Github discussions Algoritma. Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami akan menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

Best regards,
Team Algoritma
